doc/classes/CapsuleShape2D.xml

```diff
@@ -11,7 +11,10 @@
 	</tutorials>
 	<members>
 		<member name="height" type="float" setter="set_height" getter="get_height" default="30.0">
-			The capsule's height.
+			The capsule's full height, including the semicircles.
+		</member>
+		<member name="mid_height" type="float" setter="set_mid_height" getter="get_mid_height">
+			The capsule's mid height, excluding the semicircles. This is the height of the central rectangular part of the capsule, and is the distance between the centers of the two semicircles. This is a wrapper for [member height].
```

Comment:
"mid height" doesn't make much sense here. The sentence says it's "the capsule's mid height, **but** it excludes the semicircles", implying there's another `mid_height` property. Perhaps we can hint at why it's called "middle" in the later explanation.

Not sure about saying "wrapper for" but I have no concrete suggestion at the moment.
```suggestion
			The capsule's height, excluding the semicircles. This is the height of the rectangular part in the middle of the capsule in the middle, and is the distance between the centers of the two semicircles. This is a wrapper for [member height].
```